front/app/api/campaigns/__mocks__/useCampaign.ts

```diff
@@ -0,0 +1,17 @@
+export const campaignData = {
```

Comment:
```suggestion
export const campaignData: ICampaignData = {
```

This data example contains a mistake and is incomplete. Adding the expected type ensures the editor complains.

Reply:
removed the file since i removed the corresponding hook here.